cli/functionary/package.py

```diff
@@ -17,15 +17,16 @@ def create_languages() -> list[str]:
 
 
 def generateYaml(output_dir: str, name: str, language: str):
-    metadata = {
-        "name": name,
-        "version": "1.0",
-        "x-language": language,
-    }
+    package_path = pathlib.Path(output_dir).resolve() / name / "package.yaml"
+    template_path = (
+        pathlib.Path(__file__).parent.resolve() / "templates" / "template.yaml"
```

Comment:
Let's name the actual file `package.yaml`.  The fact that it's in the `templates` directory now makes it apparent that it's a template.  I suggested that in my last comment, but it probably got last in the wall of text that most of my comments end up being.